### PR TITLE
default setup answers to yes not no

### PR DESCRIPTION
**Pull request by paetling**
review @jsingle this address #389

### CODE REVIEW

**Comment by jsingle** [comment]
Not sure this is the right way to do it... I think we want a *blank* input to be the same as a Y.  Anything else is invalid (maybe should prompt for `please enter Y or N`).  I feel like that is more standard/intuitive

**Comment by paetling** [comment]
This is a good point. if they input x that should not be interpreted as a yes.   I do not think the messaging needs to change though.  I think it is more just ensuring it is y or ''

**Comment by jsingle** [comment]
OK I don't feel strongly that we need to loop input until Y or N

**Comment by jsingle** [comment]
Also you might want to make a new function which gets raw input and compares for Y or N since you're doing that in 3 places

**Comment by paetling** [comment]
ahead of you on that.  i made a new function to check for if something is a yes input

**Comment by jsingle** [comment]
:cow: :gun: :camel:

**Comment by paetling** [comment]
closes #389

File: tests/unit/commands/setup_test.py [hunk 1]
```diff
@@ -318,3 +318,8 @@ def test_get_boot2docker_vm_size_less_8_n(self, fake_virtual_memory, fake_get_ra
         self.assertEqual(_get_boot2docker_vm_size(), 1)
         fake_get_raw_input.assert_has_calls([call('Your system seems to have 6144 megabytes of memory. We would like to allocate 2048 to your vm. Is that ok? (y/n) '),
                                              call('Please input the number of megabytes to allocate to the vm: ')])
+    @patch('dusty.commands.setup._get_raw_input')
+    def test_enter_is_accepted_as_yes(self, fake_get_raw_input):
+        fake_get_raw_input.return_value = ''
+        setup_dusty_config()
```
Comment (thieman) [inline tests/unit/commands/setup_test.py]: Shouldn't you do asserts at the end of this to confirm it did what you think it should do?

**Pull request merged**